packages/babel-plugin-transform-es2015-arrow-functions/src/index.js

```diff
@@ -10,7 +10,7 @@ export default function ({ types: t }) {
           node.type = "FunctionExpression";
 
           let boundThis = t.thisExpression();
-          boundThis._forceShadow = path;
+          boundThis._forceShadow = node;
```

Comment:
True, I now still set a path, but instead the one to the functionExpressions not the callExpression. That should be okay?